src/mpi/mpi_caf.c

```diff
@@ -584,8 +585,6 @@ PREFIX (register) (size_t size, caf_register_t type, caf_token_t *token,
 
   if (type != CAF_REGTYPE_COARRAY_ALLOC_REGISTER_ONLY)
     {
-//      PREFIX(sync_all) (NULL,NULL,0);
-
```

Comment:
If there are additional changes that were not `git add`ed properly, just create a new commit with them and push your branch to github. This pull request will be automatically updated with those additional changes.